### PR TITLE
add --octave output format option for parameters

### PR DESCRIPTION
**Pull request by kd0aij**
This PR includes a new sample.ulg in test with one parameter which changes value 4 times

### CODE REVIEW

File: pyulog/params.py [hunk 1]
```diff
@@ -38,16 +41,40 @@ def main():
...
+
+            if not args.initial:
+                for t, name, value in ulog.changed_parameters:
+                    if name == param_key:
+                        values += [value]
+
+            if len(values) > 1:
+                output_file.write('\n# type: matrix\n')
+                output_file.write('\n# rows: 1\n')
+                output_file.write('\n# columns: ')
```
Comment (bkueng) [inline pyulog/params.py]: This adds 2 empty lines (after matrix & after rows). Is this intended?

Comment (kd0aij) [inline pyulog/params.py]: oops... fixed

**Comment by kd0aij** [comment]
The original sample.ulg resulted in an empty changed_parameters list, didn't it?
The one I generated has one integer and one floating point param which change values.

**Comment by kd0aij** [comment]
The new samples.ulg has an interesting "feature"; the param which I intentionally changed (using QGC) is COM_AUTOS_PAR, but MPC_Z_VEL_MAX_DN also is reported (by FlightPlot) as changing at the same time as the first two changes to COM_AUTOS_PAR, although the value remains the same.

**Comment by bkueng** [comment]
ok, squashed & applied

**Comment by bkueng** [comment]
Yes I was always wondering how accurate it is, as the logger goes through the list of unsaved parameters. It's not an error though in this case, just a bit too verbose.

**Pull request closed without merging**